server/service/src/invoice/prescription/update/validate.rs

```diff
@@ -40,6 +40,8 @@ pub fn validate(
         )
         .map_err(|e| match e {
             OtherPartyErrors::OtherPartyDoesNotExist => OtherPartyDoesNotExist {},
+            // kept this in for match but added condition so that it won't trigger for patients
+            // since you should be allowed to prescribe to a patient as long as they're on the site
```

Comment:
This comment makes sense in context of this PR but think it can be rephrased like:

// This error will only happen if the other party is not a patient.
// This is because it's allowed to prescribe to any patient on the same site.

Furthermore, please move it to the enum definition of this error instead of having a comment everywhere where it is used...